templates/common/_base/files/configure-ovs-network.yaml

```diff
@@ -17,7 +17,9 @@ contents:
       NM_CONN_PATH="$NM_CONN_UNDERLAY"
     fi
 
-    # In RHEL7 files in /{etc,run}/NetworkManager/system-connections end without the suffix '.nmconnection', whereas in RHCOS they end with the suffix.
```

Comment:
conflict: this comment was moved later in the file in a previous commit but was not cleaned up here.